incubator/redis-cache/templates/ss.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: "{{ template "fullname" . }}"
+  labels:
+     app: {{ template "fullname" . }}
+     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+     release: "{{ .Release.Name }}"
+     heritage: "{{ .Release.Service }}"
+spec:
+  serviceName: "{{ template "fullname" . }}"
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.beta.kubernetes.io/init-containers: '[
+           {
+             "name":"sentinel-micro",
+             "image":"{{ .Values.microSentinel.image }}:{{ .Values.microSentinel.tag }}",
+                         "args" : ["-service","{{ template "fullname" . }}"],
+             "env": [
+               {
+                  "name": "POD_NAMESPACE",
+                  "valueFrom": {
+                     "fieldRef" : {
+                        "apiVersion" : "v1",
+                        "fieldPath" : "metadata.namespace"
+                     }
+                  }
+               }
+             ],
+            "volumeMounts": [
+               {
+                  "name": "config",
+                  "mountPath": "/config"
+               }
+            ]
+           }
+        ]'
+    spec:
+      terminationGracePeriodSeconds: 5
+      affinity:
+           {
+              podAntiAffinity: {
+                 preferredDuringSchedulingIgnoredDuringExecution: [{
+                      weight: 5,
+                      podAffinityTerm: {
+                        labelSelector: {
+                           matchExpressions: [{
+                               "key": "app",
+                               "operator": "In",
+                                "values": ["{{ template "fullname" . }}"]
+                            }]
+                        },
+                        topologyKey: "kubernetes.io/hostname"
+                      }
+                  }]
+               }
+           }
+      containers:
+      - name: redis
+        image: {{ .Values.redis.image }}:{{ .Values.redis.tag }}
+        ports:
+        - containerPort: {{ .Values.service.port }}
```

Comment:
If the container port and service port (and) should be the same, how to we do this?